app.js

```diff
@@ -211,19 +228,96 @@ var employeeTableBody = document.getElementById('employee-table-body');
 var tableBody;
 
 printTable();
+updateStoreOption();
 
-
+////Form Submit
 var formEl = document.getElementById('form');
 formEl.addEventListener('submit', handleSubmit);
+
 function handleSubmit(event) {
   event.preventDefault();
   console.log(event.target.store.value);
   var store = event.target.store.value;
   var min = event.target.min.value;
```

Comment:
values coming from an input are strings by default, if you're expecting to use it like a number you'll need to run `parseInt` on the result.